packages/devcmd-cli/src/index.ts

```diff
@@ -73,6 +73,19 @@ async function startProcess(command: string, args: Array<string>, dirPath: strin
 async function runInDevCmdsDir(dirPath: string): Promise<void> {
   const [, , ...args] = process.argv;
 
+  if (args.length > 0) {
+    const cmdName = args[0].trim();
+
+    if (cmdName.length > 2 && cmdName.indexOf("--") === 0) {
+      try {
+        await startProcess("node", ["-e", `require('devcmd/from-cli').runReserved('${cmdName.slice(2)}')`], dirPath);
+      } catch (err) {
+        err instanceof Error ? abort(err.stack ?? err.message) : process.exit(1);
```

Comment:
I was about to ask why we don't use `abort` or emit any message when `err` isn't an `Error`, but then I saw we do the same thing when starting a regular command below 🤔

I think we should change this (in both code paths). feel free to split this to a separate ticket/PR though if you want.

Reply:
I would suggest doing this in a seperate ticket because error handling is a big thing in devcmd and can probably be improved overall!

Reply:
sounds good, please create an issue for this if there isn't one already :)